### ISSUE TITLE
Update dependencies, especially analyzer

### ISSUE THREAD

**Issue by inf0rmatix** [localizely/intl_utils#91]
This dependency lock currently produces conflicts when also using freezed package.

**Comment by aleksakrstic** [comment]
Hi @inf0rmatix 
Thanks for the info!
Can you share what version of `intl_utils` you use, and what other dependencies creates the problem for you, besides `freezed` package?

**Comment by slavap** [comment]
@aleksakrstic 
Please update analyzer, I'm also having problem with other dependencies in my projects.
For example:
```
intl_utils 2.7.0 depends on analyzer >=3.0.0 <5.0.0, intl_utils ^2.7.0 requires analyzer >=3.0.0 <5.0.0.
And because auto_route_generator >=5.0.3 depends on analyzer ^5.0.0, intl_utils ^2.7.0 is incompatible with auto_route_generator >=5.0.3.
```

**Comment by inf0rmatix** [comment]
We use the latest version of intl_utils, in our case freezed is the package that has up-to-date dependencies (on analyzer) which conflict with int_utils outdated dependencies

**Comment by lzoran** [comment]
This should be fixed with the `intl_utils 2.8.0`.
I'm closing this issue as resolved.
In case you notice any irregularity, feel free to reopen it.
